e2e/test/prerequisites/readme.md

```diff
@@ -31,3 +31,17 @@ docker run -d --restart unless-stopped --name azure-iot-tinyproxy -p 127.0.0.1:8
 Alternatives:
  - TinyProxy for Linux/Mac: https://tinyproxy.github.io/
  - Squid for Windows: https://chocolatey.org/packages/squid
+
+### ASC for IoT Security Message E2E Tests
+
+Prerequisites:
+
+- The IoT hub configured in `Configuration.IoTHub` must be onboarded to ASC for IoT security, with the feature “Store raw device security events in LogAnalytics.” set to on.
+- The tests authenticate with OMS with an Azure Active Directory app, thus an AAD app with a reader role on the OMS workspace must be created for the tests.
+
+Test configuration:
+
+- `OMS_AAD_TENANT` – The AAD tenant GUID of the OMS workspace Active Directory
```

Comment:
I don't know how to set these on my machine. After these tests get checked-in, we all need to update our dev hubs and machines. Please add detailed info on how to achieve all these, step by step, preferably with screenshots.
If such docs are already somewhere else, please ref that doc here and explain "Copy/Paste App ID from <link_with_paragraph>".